sigs.yaml

```diff
@@ -2530,6 +2530,12 @@ sigs:
     - github: michmike
       name: Michael Michael
   meetings:
+  - description: Backlog Grooming / Bug Triage Meeting
```

Comment:
Same suggestion here.

Reply:
The other files are auto-generated for this one.
Good suggestion, let me update.